scripts/mkcloud

```diff
@@ -830,17 +830,14 @@ function proposal
 function install_vbmc
 {
     # vbmc already available?
-    vbmc --version 2> /dev/null && return
-
-    # TODO: virtualbmc package is available in cloud9, switch to that,
-    #       when cloud hosts are upgraded
-    # virtualenv is also not (yet) available in standard repos
-    virtualenv --version 2> /dev/null || \
-        rpm -i http://download.suse.de/ibs/SUSE:/SLE-12-SP4:/Update:/Products:/Cloud9/standard/noarch/python-virtualenv-15.1.0-3.3.noarch.rpm
-    # include global packages to avoid compiling libvirt bindings
-    virtualenv --system-site-packages $cloud-vbmc
-    . $cloud-vbmc/bin/activate
-    pip install virtualbmc
+    vbmc --version 2> /dev/null && return 0
+
+    # some simple checks before we try to add SP4-Cloud9 repo
+    grep -q 12-SP4 /etc/os-release || return 1
+
+    # virtualbmc package is available in cloud9 repo
+    zypper ar -p 500 -f -g http://download.suse.de/ibs/SUSE:/SLE-12-SP4:/Update:/Products:/Cloud9/standard/ SLE-12-SP4-Cloud9
```

Comment:
Is `-g` = `--gpgcheck` here? There is an advantage in using long params in scripts to make them more readable.
and wouldnt -g be the default already?

Reply:
done.